src/apps/common/submodules/portWizard/portWizard.js

```diff
@@ -157,12 +165,55 @@ define(function(require) {
 
 		/**
 		 * Render the Name + Numbers step
-		 * @param  {Object} args
+		 * @param  {Object} args  Wizard args
+		 * @param  {Function} callback  Callback to pass the step template to be rendered
 		 */
-		portWizardNameAndNumbersRender: function(args) {
-			var self = this;
+		portWizardNameAndNumbersRender: function(args, callback) {
+			var self = this,
+				nameAndNumbersData = _.get(args.data, 'nameAndNumbers', {}),
+				initTemplate = function() {
+					var $template = $(self.getTemplate({
+							name: 'step-nameAndNumbers',
+							data: {
+								data: nameAndNumbersData
+							},
+							submodule: 'portWizard'
+						})),
+						$form = $template.find('form');
 
-			// TODO: Not implemented
+					monster.ui.validate($form, {
+						rules: {
+							'numbersToPort.numbers': {
+								custom: function(element) {
+									var numbersText = $(element).val();
+
+									return _
+										.chain(numbersText)
+										.split(',')
+										.map(_.trim)
+										.reject(_.isEmpty)
+										.map(monster.util.getFormatPhoneNumber)
+										.every('isValid')
+										.value();
+								}
```

Comment:
Yeah, I was wondering if it was a good solution, because even the plugin does not provide this kind of rule. `normalizer` + `phoneNumber` sounds good, I'll do it this way.